diff --git a/git-gui/git-gui.sh b/git-gui/git-gui.sh
index 201524c34e..429717c996 100755
--- a/git-gui/git-gui.sh
+++ b/git-gui/git-gui.sh
@@ -763,6 +763,15 @@ if {[is_Windows]} {
 		image create photo gitlogo32 -width 32 -height 32
 		gitlogo32 copy gitlogo -zoom 2 2
 
+		set gitlogopath "$env(HOME)/.icons/git-16.png"
+		if {[file exists $gitlogopath]} {
+		image create photo gitlogo -file $gitlogopath
+		}
+		set gitlogo32path "$env(HOME)/.icons/git-64.png"
+		if {[file exists $gitlogo32path]} {
+		image create photo gitlogo -file $gitlogo32path
+		}
+
 		wm iconphoto . -default gitlogo gitlogo32
 	}
 }
