From f33d5a527c0b2b66a1367bef48290f97a1b499c1 Mon Sep 17 00:00:00 2001
From: matt1003 <matt1003@gmail.com>
Date: Wed, 23 Apr 2025 13:13:15 +1200
Subject: [PATCH 2/3] force configured shell

---
 st.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/st.c b/st.c
index 02379c0..d2c5870 100644
--- a/st.c
+++ b/st.c
@@ -756,6 +756,8 @@ execsh(char *cmd, char **args)
 	if ((sh = getenv("SHELL")) == NULL)
 		sh = (pw->pw_shell[0]) ? pw->pw_shell : cmd;
 
+	sh = cmd; // force usage of configured shell
+
 	if (args) {
 		prog = args[0];
 		arg = NULL;
-- 
2.34.1

